mosu-app/src/entities/posts/ui/MarkDown.tsx

```
@@ -8,7 +8,7 @@ export const MarkDownRenderer = ({ content, ...props }: MarkDownRendererProps) =
     return (
         <article
             {...props}
-            className={cn(props.className, "prose [&>h1]:my-1")}
+            className={cn(props.className, "prose max-w-none [&>h1]:my-1")}
             dangerouslySetInnerHTML={{ __html: content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)